detail.html

```diff
@@ -0,0 +1,34 @@
+<%inherit file="app_base.htm" />
+
+<%block name ="content_top">
+</%block>
+
+<div id = "pagestuff">
+  <%block name = "content_middle">
+    <div id = "small_imgs">
+    <ul>
+      %for pic in product.image_urls():
+        <li class = "hover_item"><img src = "${pic}" alt = "main picture for ${product_name}"/></li>
+      %endfor
+    </ul>
+    </div>
+    <div id = "big_img">
+      <img src="${product.image_url()}" alt = "alternate picture for ${product_name}"/>
+
+    </div>
+    <div id = "detail_desc">
+    <h1>${product_name}</h1>
+
+      <p class = "prod_desc">${product.description}</p>
+    </div>
+
+    %if product.TITLE == "Bulk":
+      <form id = "q_form">
+        Quantity:
+        <input class="form-control quantityform" type = "number" value="1" name = "quantity" min = "1">
+      </form>
+    %endif:
+    <button class = "btn btn-danger" id = "buy_button">Buy Now</button>
+
+  </%block>
+</div>
```

Comment:
If you put anything outside of the block tags, I'm pretty sure they don't show up on the actual page. If you want the <div id="pagestuff"> to show up, it should be inside the <%block> tags.

Reply:
Oh, I think I just missed that /div tag